### PR TITLE
Use dummy relation in front of assignement.

### PR DESCRIPTION
**Pull request by Argc0**

### CODE REVIEW

File: test/Spec.hs [hunk 1]
```diff
@@ -96,7 +96,7 @@ sOUFFLE_DIRS = ["souffle0", -- large Doop example
                 "souffle1", "souffle2", "souffle3", "souffle4", "souffle5", "souffle6",
                  -- "souffle7", -- uses a recursive type
                  "souffle9", "souffle10", 
-                 "souffle11", "souffle12", "souffle13", "souffle14", "souffle15", "souffle16", "souffle17", "souffle18"]
+                 "souffle11", "souffle12", "souffle13", "souffle14", "souffle15", "souffle16", "souffle17", "souffle18", "souffle20"]
```
Comment (mbudiu-vmw) [inline test/Spec.hs]: where is souffle19?

File: tools/souffle_converter.py [hunk 2]
```diff
@@ -1067,7 +1067,8 @@ def process_rule(self, rule):
             convertedDisjunctions = [self.convert_terms(x) for x in terms]
             if len(convertedDisjunctions) > 0 and \
                     len(convertedDisjunctions[0]) > 0 and \
-                    convertedDisjunctions[0][0].startswith("not"):
+                    convertedDisjunctions[0][0].startswith("not") or\
```
Comment (mbudiu-vmw) [inline tools/souffle_converter.py]: We can do this for now, but in general we will need to reorder the clauses to handle many Souffle programs that we cannot compile today.

File: test/Spec.hs (hunk 1 above)
Comment (Argc0) [reply test/Spec.hs]: I am afraid that it was already measing, I did not change a thing, I only add souffle20.

Comment (mbudiu-vmw) [reply test/Spec.hs]: I have added that; will be part of a future PR.

**Pull request merged**